project/src/components/card/index.tsx

```diff
@@ -1,37 +1,40 @@
 import { Link } from 'react-router-dom';
+import { Offer } from 'mocks/offers';
 
-const id = '1';
+type cardProps = {
+  offerData: Offer;
+  onCardHover: (param: number) => void;
+  onCardClick: (param: Offer) => void;
+}
 
-function Card(): JSX.Element {
+function Card({offerData, onCardHover, onCardClick}: cardProps): JSX.Element {
   return (
-    <Link to={`/offer/${id}`}>
+    <Link to={`/offer/${offerData.id}`} onMouseOver = {() => onCardHover(offerData.id)} onClick={() => onCardClick(offerData)}>
```

Comment:
```suggestion
    <Link 
    to={`/offer/${offerData.id}`} 
    onMouseOver = {() => onCardHover(offerData.id)} 
    onClick={() => onCardClick(offerData)}
    >
```

Когда пропсов много, лучше перенести на отдельные строки, удобней читать становится

Reply:
++